<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сети теплоснабжения: Дополнительные атрибуты</string>
  </property>
  <property name="accessibleName">
   <string notr="true">btnSave</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="33" column="1">
          <widget class="QComboBox" name="cbSleeveType"/>
         </item>
         <item row="4" column="1">
          <widget class="QgsDoubleSpinBox" name="sbReturnPipeDiameter">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Диаметр обратного трубопровода, мм</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Диаметр подающего трубопровода, мм</string>
           </property>
          </widget>
         </item>
         <item row="31" column="1">
          <widget class="QComboBox" name="cbMaterialCT4"/>
         </item>
         <item row="35" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Зона теплосети</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QgsDoubleSpinBox" name="sbOccurrenceDepth">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="36" column="1">
          <widget class="QComboBox" name="cbSystemType"/>
         </item>
         <item row="31" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Материал</string>
           </property>
          </widget>
         </item>
         <item row="38" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Примечание</string>
           </property>
          </widget>
         </item>
         <item row="30" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Подземная укладка</string>
           </property>
          </widget>
         </item>
         <item row="32" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Термоизоляция</string>
           </property>
          </widget>
         </item>
         <item row="35" column="1">
          <widget class="QComboBox" name="cbHeatSupplyNetworkZone"/>
         </item>
         <item row="0" column="1">
          <widget class="QgsDoubleSpinBox" name="sbLength">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Диаметр трубопровода внешний, мм</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Протяжённость сооружения</string>
           </property>
          </widget>
         </item>
         <item row="36" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Тип система</string>
           </property>
          </widget>
         </item>
         <item row="37" column="1">
          <widget class="QComboBox" name="cbLiquidType"/>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Отметка кольца люка</string>
           </property>
          </widget>
         </item>
         <item row="19" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Отметка дна колодца</string>
           </property>
          </widget>
         </item>
         <item row="23" column="1">
          <widget class="QgsDoubleSpinBox" name="sbCrosswalkHeight">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Диаметр циркуляционного трубопровода
горячего водоснабжения, мм</string>
           </property>
          </widget>
         </item>
         <item row="21" column="1">
          <widget class="QgsDoubleSpinBox" name="sbGroundMark">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="25" column="1">
          <widget class="QgsDoubleSpinBox" name="sbGasketNumber">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QgsDoubleSpinBox" name="sbWearPercentage">
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="19" column="1">
          <widget class="QgsDoubleSpinBox" name="sbPipeBottomMark">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="32" column="1">
          <widget class="QComboBox" name="cbThermalInsulation"/>
         </item>
         <item row="34" column="1">
          <widget class="QComboBox" name="cbSleeveMaterial"/>
         </item>
         <item row="38" column="1">
          <widget class="QLineEdit" name="tbNote">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QgsDoubleSpinBox" name="sbSupplyPipeDiameterHotWater">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Глубина залегания, м</string>
           </property>
          </widget>
         </item>
         <item row="33" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Муфта тип</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QgsDoubleSpinBox" name="sbCirculationPipeDiameter">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="21" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Отметка земли</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Процент износа, %</string>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="QgsDoubleSpinBox" name="sbHatchwayRinkMark">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QgsDoubleSpinBox" name="sbPipelineOuterDiameter">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="34" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Муфта материал</string>
           </property>
          </widget>
         </item>
         <item row="30" column="1">
          <widget class="QComboBox" name="cbUndergroundLaying"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Диаметр подающего трубопровода
горячего водоснабжения, мм</string>
           </property>
          </widget>
         </item>
         <item row="37" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>Тим жидкости</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QgsDoubleSpinBox" name="sbPipeTopMark">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="29" column="1">
          <widget class="QComboBox" name="cbCounstructions"/>
         </item>
         <item row="29" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Конструкции</string>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Отметка верха трубы</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QgsDoubleSpinBox" name="sbSupplyPipeDiameter">
           <property name="maximum">
            <double>10000000.000000000000000</double>
           </property>
           <property name="clearValue">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="27" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Колодец\разрушенный или под
мощением</string>
           </property>
          </widget>
         </item>
         <item row="25" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Число прокладок</string>
           </property>
          </widget>
         </item>
         <item row="23" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Высота ароч.перехода</string>
           </property>
          </widget>
         </item>
         <item row="27" column="1">
          <widget class="QLineEdit" name="tbHatchwayState">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnOk">
         <property name="accessibleName">
          <string notr="true">btnSave</string>
         </property>
         <property name="text">
          <string>Ок</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="accessibleName">
          <string notr="true">btnCancel</string>
         </property>
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sbLength</tabstop>
  <tabstop>sbSupplyPipeDiameter</tabstop>
  <tabstop>sbReturnPipeDiameter</tabstop>
  <tabstop>sbSupplyPipeDiameterHotWater</tabstop>
  <tabstop>sbCirculationPipeDiameter</tabstop>
  <tabstop>sbPipelineOuterDiameter</tabstop>
  <tabstop>sbOccurrenceDepth</tabstop>
  <tabstop>sbWearPercentage</tabstop>
  <tabstop>sbHatchwayRinkMark</tabstop>
  <tabstop>sbPipeTopMark</tabstop>
  <tabstop>sbPipeBottomMark</tabstop>
  <tabstop>sbGroundMark</tabstop>
  <tabstop>sbCrosswalkHeight</tabstop>
  <tabstop>sbGasketNumber</tabstop>
  <tabstop>tbHatchwayState</tabstop>
  <tabstop>cbCounstructions</tabstop>
  <tabstop>cbUndergroundLaying</tabstop>
  <tabstop>cbMaterialCT4</tabstop>
  <tabstop>cbThermalInsulation</tabstop>
  <tabstop>cbSleeveType</tabstop>
  <tabstop>cbSleeveMaterial</tabstop>
  <tabstop>cbHeatSupplyNetworkZone</tabstop>
  <tabstop>cbSystemType</tabstop>
  <tabstop>cbLiquidType</tabstop>
  <tabstop>tbNote</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
